lib/GitterUser.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+var Promise = require("bluebird");
+
+var rp = require("request-promise");
+
+// miliseconds of interval between presence=online updates
+var ONLINE_PERIOD = 4 * 60 * 1000;
```

Comment:
Style: I prefer suffixing units on constants `ONLINE_PERIOD_MS` - this makes it clear that it is a `Number` and it represents milliseconds, rather than say a period "."